src/unix/stream.c

```diff
@@ -739,19 +739,6 @@ static int uv__handle_fd(uv_handle_t* handle) {
   }
 }
 
-static int uv__getiovmax() {
-#if defined(IOV_MAX)
-  return IOV_MAX;
-#elif defined(_SC_IOV_MAX)
-  static int iovmax = -1;
-  if (iovmax == -1)
-    iovmax = sysconf(_SC_IOV_MAX);
-  return iovmax;
-#else
-  return 1024;
-#endif
-}
-
```

Comment:
The getiovmax code was just moved around, since you have some issues with
it, how about we handle that in a follow-up PR?
On Jul 22, 2015 7:20 AM, "Ben Wijen" <notifications@github.com> wrote:

> In src/unix/stream.c
> <https://github.com/libuv/libuv/pull/448#discussion_r35182279>:
>
> > @@ -739,19 +739,6 @@ static int uv__handle_fd(uv_handle_t* handle) {
> >    }
> >  }
> >
> > -static int uv__getiovmax() {
> > -#if defined(IOV_MAX)
> > -  return IOV_MAX;
> > -#elif defined(_SC_IOV_MAX)
> > -  static int iovmax = -1;
> > -  if (iovmax == -1)
> > -    iovmax = sysconf(_SC_IOV_MAX);
> > -  return iovmax;
> > -#else
> > -  return 1024;
> > -#endif
> > -}
> > -
>
> You also want to define __need_IOV_MAX before the limits.h inclusion...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/libuv/libuv/pull/448/files#r35182279>.
>